### PR TITLE
Build server

### PR DESCRIPTION
**Pull request by Gr33ny**
Automated build server setup.

### CODE REVIEW

File: concent-builder-vm/readme.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# HOW TO SETUP A VAGRANT MACHINE
```
Comment (cameel) [inline concent-builder-vm/readme.txt]: The instructions for installing Vagrant, VirtualBox, searching for a box on the web page, downloading it, etc. are not needed.

Installing dependencies is just a matter of installing the `vagrant` package and instructions for doing that are system dependent. We can safely assume that a competent developer will know how to install a package in his own system. And if not, it's not in the scope of our project scope to try to explain it.

Vagrant box is already hard-coded in your `Vagrantfile` so no need to explain the search process either. The box will be downloaded automatically on the first run. It's all a matter of a single instruction - `vagrant up`.

So, in summary, it's enough to just mention in the main README file that `build-server-vm` contains configuration script for a virtual machine, that it depends on Vagrant and that you create it by running `vagrant up`. That's it. Please remove the commit with these instructions. You can put them on Github wiki if you really want to keep it.

Comment (cameel) [inline concent-builder-vm/readme.txt]: Use Markdown formatting for text files like this.

File: concent-builder-vm/readme.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+3. Install VirtualBox provider inside Vagrant. 
+COMMAND:
+
+vagrant box add \
+  precise64 https://files.hashicorp.com/precise64.box
+
+4. Install image your machines. 
+Choose from: https://app.vagrantup.com/boxes/search?_ga=2.138139312.1235557729.1509978713-689630316.1509978713
```
Comment (cameel) [inline concent-builder-vm/readme.txt]: What's `_ga=2.138139312.1235557729.1509978713-689630316.1509978713` for? Is it some kind of tracking ID (does not seem to affect search results)? Please make sure you never commit stuff like this into the repository.

File: concent-builder/configure.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            - google-cloud-sdk
+            - ansible
+            - virtualenv
+
+        - name:    apt autoclean
+          command: apt autoclean
+        - name:    apt autoremove
+          command: apt autoremove
+
+        - name: Make a Docker group
```
Comment (cameel) [inline concent-builder/configure.yml]: Creating the `docker` group is not necessary. It should be created by docker automatically.

File: concent-builder/configure.yml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          group:
+            name:  docker
+            state: present
+
+        - name: Add a User to group
```
Comment (cameel) [inline concent-builder/configure.yml]: `ansible_user` is the user running the playbook. If you use that name to create an account, different people running the playbook will create different accounts. This was not the intention. This was meant to create a common user account with a fixed name that everyone will be able to use to build and deploy.

File: concent-builder/configure.yml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+          user:
+            name:   "{{ ansible_user }}"
+            state:  present
+            groups: docker
+
+        - name: Install Yasha module
```
Comment (cameel) [inline concent-builder/configure.yml]: Let's install this module globally, not in a virtualenv.

**Comment by cameel** [comment]
The problems have all been fixed. I've merged the branch into master.
